src/sections/Meshery/Features-section/index.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { Row, Col } from "../../../reusecore/Layout";
+import Button from "../../../reusecore/Button";
+import meshery_logo from "../../../assets/images/meshery/meshery-logo-tag-light-white-text-side.png";
+import Slider from "react-slick";
+import slide1 from "../images/meshery_benchmark_screen.png";
+import slide2 from "../images/meshery-configuration-management.png";
+import slide3 from "../images/SMI-Conformance-in-Meshery.png";
+import FeaturesSectionWrapper from "./features-section.style";
+
+const slides_list = [slide1, slide2, slide3];
+
+const FeaturesSection = () => {
+
+  var settings = {
+    customPaging: function(i) {
+      return (
+        <a>
+          <img src={slides_list[i]} />
+        </a>
+      );
+    },
+    infinite: false,
+    speed: 400,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    dots: true,
+    vertical: true,
+    verticalSwiping: true
+  };
+
+  return (
+    <FeaturesSectionWrapper>
+      <div className="skw"></div>
+      <div>
+        <div className="rect-1-left"></div>
+        <div className="rect-2-left"></div>
+        <div className="rect-3-right"></div>
+        <div className="rect-4-right"></div>
+      </div>
+      <div className="section-data">
```

Comment:
This entire part should be inside `Container`?

Reply:
@Tanuj22, you're right that the design has this in a container. I think it's working well outside of a container. If you're ok with it not being contained, so am I. ✔️

Reply:
Perhaps, let's merge in order to support going live with a potential follow-up issue if it turns out that the container less design isn't working later?

Reply:
Yeah sure

Reply:
The reason for not using container was to give some breathing space to the carousel and the features text having side by side....else it will get very conjested....so I didn't use the container instead added the `max-width` for them individually.